src/EditorFeatures/Core/Implementation/Diagnostics/AbstractAggregatedDiagnosticsTagSource.cs

```diff
@@ -52,6 +53,11 @@ protected override void Disconnect()
 
         public override ITagSpanIntervalTree<TTag> GetTagIntervalTreeForBuffer(ITextBuffer buffer)
         {
+            return GetAccurateTagIntervalTreeForBuffer(buffer, CancellationToken.None);
```

Comment:
Yes, this makes it the same as before for all AbstractAggregatedDiagnosticsTagSource implementations.